src/modules/agents/ui/components/agents-search-filter.tsx

```
@@ -0,0 +1,26 @@
+import React, { useState } from "react";
+import { useAgentsFilters } from "../../hooks/use-agents-filters";
+import { Input } from "@/components/ui/input";
+import { SearchIcon } from "lucide-react";
+
+type Props = {};
```

Biome (1.9.4)

[error] Don't use '{}' as a type.

Prefer explicitly define the object shape. '{}' means "any non-nullable value".

(lint/complexity/noBannedTypes)